### PR TITLE
Roboarmcontroller intro post

### PR DESCRIPTION
**Pull request by ellerenad**

### CODE REVIEW

File: _posts/2020-04-24-roboarmcontroller-intro.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "RoboArm Controller, part I: Intro and architecture."
```
Comment (FaKleiser) [inline _posts/2020-04-24-roboarmcontroller-intro.md:3]: Titles should have consistent capitalization to improve readability. I usually use this tool for the job (choose and stick to one of the conventions throughout the blog): https://capitalizemytitle.com

File: _posts/2020-04-24-roboarmcontroller-intro.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+This is just an overview of each of the main components used for the project. If these series get enough traction, I will
+write a post describing each one of them
+
+### Hands' sensor
+
+This is the origin of the data, where the position of the hands is sensed.
+Used the [Leap Motion][leap_motion] sensor. It has a framework to communicate with different languages, but the way I got to talk 
+with it was through a WebSocket. It provides coordinates for a set of points related to the hands, e.g. the tips of the fingers,
+the center of the hand, and so on.
+- Technologies used: Leap Motion.
```
Comment (FaKleiser) [inline _posts/2020-04-24-roboarmcontroller-intro.md:88]: Can you add a link?

File: _posts/2020-04-24-roboarmcontroller-intro.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+- Interfaces: Websocket.
+- Input: The hands.
+- Output: Position of different parts of the hands, sent to the data transformer application.
```
Comment (FaKleiser) [inline _posts/2020-04-24-roboarmcontroller-intro.md:91]: As it's a classification problem, the output should be a rather simple data model? E.g. a 3-dimensional vector with an entry for each servo?  Maybe that is quicker/easier to understand than the text?

File: _posts/2020-04-24-roboarmcontroller-intro.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+- The machine learning part is on the data transformation: classify the position of the hands to get a movement instruction.
+- The RoboArm Controller covers the machine learning cycle: Gather the data, prepare the data, choose a model, train the model,
+ tune the hyperparameters, test the model, export the model, import the model into a "production" system, and use it to make predictions.
+- The project was presented at the [Java Forum Stuttgart 2019][JFS_roboarmcontroller] and the EclipseCon Europe 2019 ([video][eclipsecon_roboarmcontroller_video]/[slides][eclipsecon_roboarmcontroller_slides])
+- The RoboArm Controller is a hobby project I developed to practice different technologies in a ludic way.
+
+
+## Introduction
+
+The RoboArm Controller allows controlling a simulated robotic arm with the position of the hands of a person.
```
Comment (FaKleiser) [inline _posts/2020-04-24-roboarmcontroller-intro.md:35]: The intro is a bit quick, the problem description starts too soon for my feeling. Spend 1-2 more sentenes describing the setup, e.g. that you have 3 servos on the arm that you want to control, etc. That makes understanding the following sentences easier (for those who don't have the project context)

**Review by FaKleiser — COMMENTED** [review]
Nice article, and also good sectioning with the headlines, keeping the sections concise and to the point 👍

File: _posts/2020-04-24-roboarmcontroller-intro.md (hunk 3 above)
Comment (ellerenad) [reply _posts/2020-04-24-roboarmcontroller-intro.md:91]: this comment applies to the output of the data transformer application. Added an example of the output there

**Comment by ellerenad** [comment]
> Nice article, and also good sectioning with the headlines, keeping the sections concise and to the point 👍

applied the feedback, thank you very much for the review, @FaKeller !

**Pull request merged**